flow/CompressionUtils.cpp

```diff
@@ -25,24 +25,16 @@
 #include "flow/IRandom.h"
 #include "flow/UnitTest.h"
 
-#include <boost/iostreams/copy.hpp>
-#ifdef ZLIB_LIB_SUPPORTED
-#include <boost/iostreams/filter/gzip.hpp>
-#endif
-#include <boost/iostreams/filtering_streambuf.hpp>
 #ifdef ZSTD_LIB_SUPPORTED
-#include <boost/iostreams/filter/zstd.hpp>
```

Comment:
If you're removing all of the ZLIB stuff should we just fully rip it out, including out of the build? We could also do that as a separate PR

Reply:
yeah.. plan to do the cleanup in a separate PR.

Reply:
+1 we should remove ZLIB from code and CMake files

Reply:
thank you.. will do it in next PR